### PR TITLE
Add default contact request

### PR DESCRIPTION
**Pull request by cammellos**
If a contact update or a legacy contact request is sent, we create a
notification for the user in the activity center so it can be replied
to.
If at a later date a new contact request is received from the same user,
this will replace it, so the proper message can be displayed.

### CODE REVIEW

File: protocol/messenger_handler.go [hunk 1]
```diff
@@ -631,6 +669,7 @@ func (m *Messenger) HandlePinMessage(state *ReceivedMessageState, message protob
 }
 
 func (m *Messenger) HandleAcceptContactRequest(state *ReceivedMessageState, message protobuf.AcceptContactRequest) error {
+	m.logger.Info("HANDLING")
```
Comment (Samyoul) [inline protocol/messenger_handler.go]: Is this a left over from development?

File: protocol/messenger_handler.go [hunk 2]
```diff
@@ -738,15 +777,14 @@ func (m *Messenger) HandleContactUpdate(state *ReceivedMessageState, message pro
 		contact.LastUpdated = message.Clock
 		state.ModifiedContacts.Store(contact.ID, true)
 		state.AllContacts.Store(contact.ID, contact)
-		/* Disabling for now in order to avoid duplicated contact requests in activity center
```
Comment (Samyoul) [inline protocol/messenger_handler.go:741]: Perhaps add a `TODO` for removing or reactivating when the right circumstances have been met?

**Review by Samyoul — APPROVED** [review]
Lovely PR, nice comments

File: protocol/messenger_handler.go (hunk 1 above)
Comment (cammellos) [reply protocol/messenger_handler.go]: yes :)

File: protocol/messenger_handler.go (hunk 2 above)
Comment (cammellos) [reply protocol/messenger_handler.go:741]: wait, this is a removal, or I misunderstand your comment? i.e, I have now removed the comment and enabled the feature

Comment (Samyoul) [reply protocol/messenger_handler.go:741]: OH, I'm so slow! I completely misread the diff. Ignore this 🤦

**Review by richard-ramos — APPROVED** [review]
:100:

**Pull request merged**